index.html

```diff
@@ -112,42 +112,41 @@ <h3 class="item item-header dark-orange">Lecture</h3>
           <img src="image/program_icon_02.png" class="img-fluid rounded-start" alt="...">
         </div>
         <h3 class="item item-header dark-orange">Exhibition</h3>
-        <p class="item item-content light-grey">
+        <p class="item item-content ">
           Taking place virtually, we are offering you an opportunity <br> to be featured on the official Exhibition.
         </p>
       </article>
       <article class="program-item flex-row-center bg-light-black">
         <div class="item item-icon heading-big">
-          <img src="image/program_icon_03.png" class="img-fluid rounded-start" alt="...">
+          <img src="image/program_icon_03.png" class="img-fluid " alt="...">
         </div>
         <h3 class="item item-header dark-orange">Forum</h3>
-        <p class="item item-content light-grey">
+        <p class="item item-content ">
           We are proud to have joined the Africa No Filter network as <br> narrative champions.
 
         </p>
       </article>
       <article class="program-item flex-row-center bg-light-black">
         <div class="item item-icon heading-big">
-          <img src="image/program_icon_04.png" class="img-fluid rounded-start" alt="...">
+          <img src="image/program_icon_04.png" class="img-fluid " alt="...">
         </div>
         <h3 class="item item-header dark-orange">Workshop</h3>
-        <p class="item item-content light-grey">
+        <p class="item item-content ">
           We are excited to open this application to you and <br> more African podcasters.
-        </p>
       </article>
       <article class="program-item flex-row-center bg-light-black">
         <div class="item item-icon heading-big">
           <img src="image/program_icon_05.png" class="img-fluid rounded-start" alt="...">
         </div>
         <h3 class="item item-header dark-orange">Ignite</h3>
-        <p class="item item-content light-grey">
+        <p class="item item-content ">
```

Comment:
- Kindly make sure your program section stick to Cindy's design to meet the project requirements:
  - Your design:
![image](https://user-images.githubusercontent.com/97834160/233808495-c833ad12-b24e-4cac-992d-165c7d43ea7d.png)

  - Cindy's design:
![Screenshot from 2023-04-23 00-07-02](https://user-images.githubusercontent.com/97834160/233808416-2e8b997e-0950-4f04-bfcc-449eb050f141.png)